package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sequelize-cli-helpers",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
this looks great - but don't think we need to bump the version for just a test! Please fix.